src/main/scala/cromwell/engine/backend/local/LocalBackend.scala

```diff
@@ -13,21 +13,35 @@ import scala.util.Try
 class LocalBackend extends Backend {
 
   override def executeCommand(commandLine: String, call: Call, taskOutputs: Set[TaskOutput], symbolStore: SymbolStore): Map[String, Try[WdlValue]] = {
+
+    implicit class FlushingAndClosingWriter(writer: Writer) {
```

Comment:
It seems like something which belongs in an Object anyways, but I don't know the specific objection the compiler has